server_action_wizard/models/ir_actions_server.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class IrActionsServer(models.Model):
+    _inherit = "ir.actions.server"
+
+    can_run_from_wizard = fields.Boolean("Can run from wizard")
```

Comment:
Is it possible to have something configurable ?

I see that 4 fields quite custom...

Reply:
The 4 fields are for technical simplicity. It would be easy to add more if that makes sense.
I don't see how it even more generic.

Reply:
It will display fields in the wizard, even if it doenst make sense. 

I dont see it very user friendly, or did I miss something ? Could you provide use case with real exemple in the readme section ? (Based on démo data could be great !)

> I don't see how it even more generic.

Using fields.Properties could make the job I guess.

Thanks !